core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/WorkspaceMigrationMonitor.scala

```diff
@@ -86,55 +100,95 @@ object WorkspaceMigrationMonitor {
     )
 
     for {
-      googleProject <- attempt.newGoogleProjectId match {
-        case Some(projectId) => IO.pure(GoogleProject(projectId.value))
-        case None            => failNoGoogleProject(attempt)
-      }
+      googleProjectId <- IO(attempt.newGoogleProjectId.getOrElse(throw noGoogleProjectError(attempt)))
 
       _ <- createBucketInSameRegion(
         GcsBucketName(workspace.bucketName),
-        googleProject,
+        GoogleProject(googleProjectId.value),
         tmpBucketName,
         googleStorageService
       )
 
-    } yield (tmpBucketName, DBIO.seq(
-      workspaceMigrations
-        .filter(_.id === attempt.id)
-        .map(r => (r.tmpBucket, r.tmpBucketCreated))
-        .update((tmpBucketName.value.some, Timestamp.valueOf(LocalDateTime.now).some))
-    ))
+      created <- timestampNow
+
+    } yield (tmpBucketName, workspaceMigrations
+      .filter(_.id === attempt.id)
+      .map(r => (r.tmpBucket, r.tmpBucketCreated))
+      .update((tmpBucketName.value.some, created.some))
+      .ignore
+    )
   }
 
+  final def deleteWorkspaceBucket(migration: WorkspaceMigration,
+                                  workspace: Workspace,
+                                  googleStorageService: GoogleStorageService[IO]
+                                 ): IO[WriteAction[Unit]] =
+    for {
+      msuccess <- googleStorageService.deleteBucket(
```

Comment:
`msuccess`?

Reply:
This is a common fp naming scheme. `success` is boxed in a monad, so `m` is offen prefixed to denote this (ie it's not a bool, it's an m[Bool])

Reply:
As someone who isn't as familiar with this naming convention, I would personally rather either explicitly spell it out (`monadSuccess`), or let the automatic typechecker tell us what it is and leave it as `success`. To me atm it reads as an accidental keystroke, which it sounds like its not. If we really want this naming convention in the code, how do you feel about `mSuccess`? That way its at least not reading as an accidental keystroke, because of the camelcase.